portfolio/src/main/java/com/google/sps/servlets/CommentProcessorServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+
+/** Servlet that processes comment. TODO: Improve formatting of response output*/
```

Comment:
Please be more specific in javadoc about the servlet.  E.g, it has post and get requests, what does each do, and what does each return.

Reply:
right added more details :)